### ISSUE TITLE
Add auth tokens to /me endpoint

### ISSUE THREAD

**Issue by juanignaciosl** [CartoDB/cartodb#13121]
_TBD_

As `/me` is safe, we could always add them. Would that work, @elenatorro ?

**Comment by elenatorro** [comment]
Yes, I think so. I will add them then to the api in the affected PR (https://github.com/CartoDB/cartodb/pull/13043), and then will ask someone from backend to review it :) thank you very much!

**Comment by juanignaciosl** [comment]
Assigned to you, then!

**Comment by elenatorro** [comment]
Hmm I have been thinking that maybe, as this modification has been done in `admin/visualizations_controller`, we should return it only when we request a visualization in `api/visualizations_controller`. I don not know why would we need it when we get the user's info. What do you think @juanignaciosl ?

**Comment by juanignaciosl** [comment]
Oh, when I did a quick look at this I just saw [the ones requested at user level](https://github.com/CartoDB/cartodb/blob/4415b533a9fad5835fa7e6c11b5d4611ff06daf4/app/controllers/admin/visualizations_controller.rb#L153), but it looks like you meant [the visualization level ones](https://github.com/CartoDB/cartodb/blob/4415b533a9fad5835fa7e6c11b5d4611ff06daf4/app/controllers/admin/visualizations_controller.rb#L361).

I just updated the issue description.

How does it sound?
